frontend/pages/events/[id].tsx

```diff
@@ -118,6 +125,79 @@ const Divider = styled.hr`
   margin: 20px 0;
 `
 
+const Input = styled.input`
+  border: 1px solid ${ALLBIRDS_GRAY};
+  outline: none;
+  color: ${CLUBS_GREY};
+  flex: '0 0 auto';
+  font-size: 1em;
+  padding: 8px 10px;
+  margin: 0px 5px 0px 0px;
+  background: ${WHITE};
+  border-radius: ${BORDER_RADIUS};
+  min-width: 50px;
+  font-family: ${BODY_FONT};
+  &:hover,
+  &:active,
+  &:focus {
+    background: ${FOCUS_GRAY};
+  }
+`
+
+type Ticket = {
+  type: string
+  price: string
+  max: string
+  count: string | null
+}
+
+const TicketItem = ({
+  ticket,
+  name,
+  price,
+  max,
+  changeCount,
```

Comment:
convention: use `on...` for sending events to parent.
also try not to rely on index as the children should barely be aware of its own index(it's better to delegate this job to the parent and make the parent take action as it's guaranteed to have context about each item's index)